specs/progress.md

```diff
@@ -0,0 +1,150 @@
+# Progress
+
+## Overview
+
+*Progress* is used to display the length of time a process will take (referred to as a **determinate** state) or to represent an unspecified wait time (referred to as an **indeterminate** state). *Progress* is typically visually represented by a circular or linear animation. When the `value` attribute is passed the state is **determinate**, otherwise it is **indeterminate**.
+
+### Use Cases
+
+The *progress* component should be used for an element that displays the progress status for a task that takes a more than a few seconds or consists of several steps. Typical use cases include loading elements or status of a download. 
```

Comment:
Rather than what it shouldn't do, can we add some use cases for when someone may be trying to use it and what they are doing with it? I added a few for Dialog like this.

Reply:
Sorry, I'm confused by this. Where do I say what authors **should not** do with this component?

Reply:
Updated, I must have read this wrong 🤦‍♂